doc/R.nvim.txt

```diff
@@ -727,8 +727,8 @@ You may close the window opened by `:RConfigShow` by simply pressing `q`.
 |assignment_keymap|   Choose what to convert into ' <- '
 |pipe_keymap|         Choose what to convert into ' |> ' (or ' %>% ')
 |pipe_version|        Either "native"/"|>" or "magrittr"/"%>%"
-|rnowebchunk|         Convert '<' into '<<>>=\n@' in Rnoweb files
-|rmdchunk|            Convert grave accent chunk into delimiters
+|rnw_chunk_keymap|    Choose what to convert to R chunk delimiters
+|rmd_chunk_keymap|    Choose what to convert to R chunk delimiters
```

Comment:
I am not sure to understand what this means. Do you mean something like:

Which key map to insert inline or R code blocks?

Reply:
Here we are talking about which keymap right? Not the character to be used like "`" or something else?